eng/SourceBuild.Tarball.targets

```diff
@@ -320,12 +320,14 @@
     <MSBuild
       Projects="$(LeakDetectionProjectFile)"
       Targets="Restore"
-      Properties="RestoreSources=$(SourceBuiltPackagesPath)%3B$(TarballRootDir)packages/prebuilt%3bhttps://api.nuget.org/v3/index.json"/>
+      Properties="RestoreSources=$(SourceBuiltPackagesPath)%3B$(TarballRootDir)packages/prebuilt%3bhttps://api.nuget.org/v3/index.json
+                  DotNetPackageVersionPropsPath=$(PackageVersionPropsPath)"/>
```

Comment:
Don't these still need the `;`? Also... not quite my suggestion (I prefer not using alignment) but that's less important 😛:

```
      Properties="
        RestoreSources=$(SourceBuiltPackagesPath)%3B$(TarballRootDir)packages/prebuilt%3bhttps://api.nuget.org/v3/index.json;
        DotNetPackageVersionPropsPath=$(PackageVersionPropsPath)"/>

```